internal/controllers/device_config_controller.go

```diff
@@ -54,6 +56,11 @@ type DeviceConfigResponse struct {
 	Version          string `json:"version"`
 }
 
+type UserDeviceTemplateResponse struct {
```

Comment:
let's rename this to just be DeviceTemplateResponse, and the service too, eg. deviceTemplateService